controllers/ycsbbench/controller.go

```diff
@@ -0,0 +1,77 @@
+package ycsbbench
```

Comment:
Can you please add apache header to the beginning of all the go source files?

Reply:
Done!